src/libs/actions/ReportActions.ts

```diff
@@ -58,18 +77,6 @@ function clearReportActionErrors(reportID: string, reportAction: ReportAction, k
     });
 }
 
-let allReportActions: OnyxCollection<ReportActions>;
-Onyx.connect({
-    key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
-    waitForCollectionCallback: true,
-    callback: (actions) => {
-        if (!actions) {
-            return;
-        }
-        allReportActions = actions;
```

Comment:
> Is there any reason this isn't like the other connections with (value) => (allReportActions = value),?

@tgolen I just move this to above.

Reply:
OK, would you mind updating it to be a simple callback like the others? It just helps to clean it up a tad.